ehrapy/tools/_sa.py

```diff
@@ -2,11 +2,12 @@
 
 from typing import TYPE_CHECKING, Literal
 
+import anndata as ehrapy_ad
```

Comment:
I don't see how this import makes sense.

The `anndata_to_df` function is within ehrapy itself (anndata_ext). Please use Pycharm or something else and import this properly